localization/strings_en-US.json

```diff
@@ -211,6 +211,47 @@
   "command_permitremind_success": "Successfully added the permission.",
   "command_permitremind_duplicate": "Such permission already exists. Skipped.",
 
+  "command_makeimage_name": "makeimage",
```

Comment:
command_makeimage_help is missing both in en-US and ru-RU locales. Please add it and put some example of usage there (e.g. the text you used while testing the command).